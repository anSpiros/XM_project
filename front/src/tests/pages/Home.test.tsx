import React from "react";
import { render, screen } from "@testing-library/react";
import HomePage from "../../pages/HomePage";

describe("Home page component", () => {
  test("should render correctly", () => {
    const { asFragment } = render(<HomePage />);

    expect(asFragment()).toMatchSnapshot();
  });

  test("should contain the word Welcome", () => {
    render(<HomePage />);

    expect(screen.getByText(/Welcome/i)).toBeInTheDocument();
  });
});
