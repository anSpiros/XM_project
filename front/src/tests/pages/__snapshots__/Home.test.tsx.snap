// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Welcome
    </h1>
  </div>
</DocumentFragment>
`;
